net/portmapper/portmapper.go

```diff
@@ -714,7 +714,10 @@ func (c *Client) Probe(ctx context.Context) (res ProbeResult, err error) {
 				res.UPnP = true
 				c.mu.Lock()
 				c.uPnPSawTime = time.Now()
-				c.uPnPMeta = meta
+				if c.uPnPMeta != meta {
+					c.logf("[upnp-stats] meta changed: %+v", meta)
```

Comment:
`UPnP meta changed: %+v` is fine.

I'm not a fan of the pretending-we-have-structured-logging with [foo] annotations, at least when they're one-off cases.